src/components/VolAttendanceListItem/index.tsx

```diff
@@ -33,20 +60,41 @@ const VolAttendanceListItem: React.FC<Props> = ({ eventId, eVol }) => {
                 alert(`Error: ${response.message || "Unexpected error."}`);
             }
         }
+        setLoading(false);
     };
     return (
         <React.Fragment>
-            <TableCell>{eVol.volunteer.name}</TableCell>
-            <TableCell align="right">
-                <Switch
-                    onClick={toggleAttendance}
-                    checked={present}
-                    name="presentSwitch"
-                    inputProps={{ "aria-label": "secondary checkbox" }}
-                />
+            <TableCell>
+                <CoreTypography variant="body1">{eVol.volunteer.name}</CoreTypography>
             </TableCell>
+            <TableCell align="right">{getCheckBox()}</TableCell>
         </React.Fragment>
     );
 };
 
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        checkedBox: {
+            backgroundColor: colors.green,
+            border: "2px solid black",
+            width: "50px",
+            height: "64px",
+            borderRadius: "50%",
+            "&:hover": {
+                backgroundColor: colors.green,
+            },
+        },
+        uncheckedBox: {
+            backgroundColor: colors.white,
+            border: "2px solid black",
+            width: "50px",
+            height: "64px",
+            borderRadius: "50%",
+            "&:hover": {
+                backgroundColor: colors.white,
+            },
+        },
+    })
+);
+
```

Comment:
i like the changes here, was wondering how to make a loading thing. im gonna pull this down and merge it w my fixes to make sure its all good